### ISSUE TITLE
After i created a private repository on github, unable to link and open the repository in bolt.new

### ISSUE THREAD

**Issue by KnolBee** [stackblitz/webcontainer-core#1552]
### Describe the bug

I created a repository on github, named it differently while creating. There is no way i can open the same repository in bolt.new now. The option just doesnt show up.

### Link to the blitz that caused the error

https://stackblitz.com/~/github.com/KnolBee/TradingConsole?file=.bolt/config.json

### Steps to reproduce

1. Create a new project in bolt.new
2. Open the source in stackblitz
3. Create repository on github and name it friendly
4. No way the new repository can be linked with bolt.new

### Expected behavior

I should be able to edit the respository in bolt.new

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Screenshots

NA

### Platform

```
Version = 1.84.0
Hash = 3b0b7194bc6b567c2eaf4c96a5759098469dcf59
WebContainer = 34c588ed

Browser name  = Chrome
Full version  = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 91897092,
  "usedJSHeapSize": 86306728,
  "jsHeapSizeLimit": 4294705152
}
```

### Additional context

NA

**Comment by HenkDz** [comment]
same issue. I can't link an existing project with GitHub repo as well. So basically bolt wouldn't link to GitHub directly nor through the Projects.

**Comment by HenkDz** [comment]
This also should have the LINK option

![image](https://github.com/user-attachments/assets/0a39e405-c8bf-486a-8d5d-d528c7d65d47)

**Comment by invenics-sourav** [comment]
i am having the same problem, it creates the repo on my git account but the project code is not updated. it has the read-write permission still not working.

**Comment by etaytch** [comment]
Same issue here!

**Comment by har1mau** [comment]
Same issue for me. Cannot come back to something after a day or two and resume working on it in bolt.new.

**Comment by the-digital-pro** [comment]
+1 At this moment it feels only one direction from bolt to stackblitz. Once I have my project in a repo I would like to resume the prompting in bolt and then commit changes back to the repo from stackblitz.

**Comment by nlib** [comment]
+1 majorly important to be able to keep working on the project in bolt, once exported and modified in stackblitz. How can this be done?

**Comment by alex-symphonic** [comment]
agreed with above! It would be nice to have bolt more tightly integrated/available in stackblitz codeflow. I currently need to go back and add a modal, it would be awesome to pop open the repo in stackblitz and prompt bolt to add it.
